keras_cv/models/backbones/resnet_v1/resnet_v1_backbone_presets_test.py

```diff
@@ -63,10 +59,9 @@ def test_backbone_output_with_weights(self):
         # We should only update these numbers if we are updating a weights
         # file, or have found a discrepancy with the upstream source.
 
-        outputs = feature_extractor(tf.ones(1, 512, 512, 3))
+        outputs = feature_extractor(tf.ones((1, 512, 512, 3)))
         outputs = [outputs[i][0, 0, 0, :5] for i in extractor_levels]
```

Comment:
I think this test deviated from the resnet_v2 version by mistake. That is doing what we want ([link](https://github.com/keras-team/keras-cv/blob/master/keras_cv/models/backbones/resnet_v2/resnet_v2_backbone_presets_test.py#L29))

Reply:
This test is more comprehensive since it tests all feature pyramid outputs. There is no other difference from the resnet v2 tests.

Reply:
I'm not sure that's necessary honestly. What would it catch that a forward pass would not?

Just don't want to add unnecessary overhead for our model development.

Reply:
@LukeWood may provide some comments here. It is requested in [this comment](https://github.com/keras-team/keras-cv/pull/1508#discussion_r1152394503). I am fine with either way.

Reply:
This test would be better as a RetinaNet forward pass test rather than requiring every backbone test to write out 15 floats.

Reply:
I'm fine either way here -- but as-is this test is broken due to this call to `tf.ones`, so it's probably worth fixing.

Reply:
Done. Changed to only testing the final output without feature extractor.